src/main/java/com/amashchenko/maven/plugin/gitflow/AbstractGitFlowMojo.java

```diff
@@ -1188,7 +1224,26 @@ private void executeGitCommand(final String... args)
      */
     private void executeMvnCommand(final String... args)
             throws CommandLineException, MojoFailureException {
-        executeCommand(cmdMvn, true, argLine, args);
+
+        initMavenDefaults();
+
+        final List<String> argLines = new ArrayList<String>();
+
+        if (StringUtils.isBlank(this.argLine)
+                || !this.argLine.contains("-gs")) {
```

Comment:
there are multiple variants of this parameter - probably check both `-gs` and `--global-settings`

Reply:
Going to change code, if they define the new custom variables then simply put them 1st. So won't need to check argLine.